ord_schema/message_helpers_test.py

```diff
@@ -18,13 +18,11 @@
 
 import pandas as pd
 import pytest
-from google.protobuf import json_format
-from google.protobuf import text_format
+from google.protobuf import json_format, text_format
```

Comment:
same comment here about importing modules on different lines

Reply:
Fixed in cf20d57.